src/emit.iced

```diff
@@ -60,22 +69,39 @@ exports.GoEmitter = class GoEmitter
   tab : () -> @_tabs++
   untab : () -> @_tabs--
 
+  make_map_type : ({t}) ->
+    key = if t.keys? then @emit_field_type(t.keys).type else "string"
```

Comment:
We only need `go_lint_capitalize` for types that are generated implicitly for RPCs, like `void pgpImport()` let's say.  So not needed here unless we make deep copiers for those guys (which can do!).